lib/shopify-cli/git.rb

```diff
@@ -1,3 +1,5 @@
+require 'open3'
```

Comment:
You should probably just require this on line 126 to avoid cli slowdown

Reply:
Absolutely. Moved to the usage point.